Objective-C/Tests/CBLTestCase.m

```diff
@@ -90,6 +91,10 @@ - (void) tearDown {
     [super tearDown];
 }
 
++ (void) tearDown {
+    [CBLLogSinks setVAPI: LogAPINone];
```

Comment:
here to reset after each test class run. So it doesn't crash when running in pipeline.

Reply:
In the change I suggested, it will call `+ (void) resetApiVersion` and if there is a crash, let look at that and fix it.

Reply:
I received a crash when setting to nil on test tearDown, as expected. I will gatekeep against and check